recipe/meta.yaml

```diff
@@ -22,7 +23,8 @@ requirements:
 
   run:
     - python
-    - google-cloud-core >=0.27.0,<0.28dev
+    - google-cloud-core >=0.29.0,<0.29dev
```

Comment:
This may be a typo. I notice https://pypi.python.org/pypi/google-cloud-bigquery/0.28.0 has a requirement of `google-cloud-core (>=0.28.0,<0.29dev)`

Reply:
You're right. Thanks.